src/utils/session-core.js

```diff
@@ -284,17 +448,17 @@ export const deleteDocumentContainer = async (session, documentUrl) => {
  */
 
 export const generateUsersList = async (session) => {
-  const userContainerUrl = getContainerUrl(session, 'none', 'self-fetch');
+  const userContainerUrl = getContainerUrl(session, 'Users', 'self-fetch');
```

Comment:
Left a comment earlier about 'self-fetch', but we should consider combining 'self-fetch' and 'cross-fetch' as part of constants.

Reply:
Related to Issue #161